config/routes.rb

```diff
@@ -7,4 +7,5 @@
   }
 
   resources :users
+  resources :roles
```

Comment:
do we need all the actions? why would we need update or index or even show? do you have a roles controller and do you need these routes?